outline/electron/main.go

```diff
@@ -100,8 +100,24 @@ func main() {
 		os.Exit(oss.IllegalConfiguration)
 	}
 
+	ssclient, err := client.NewClient(*args.proxyHost, *args.proxyPort, *args.proxyPassword, *args.proxyCipher)
```

Comment:
This is duplicated code.
Should we start passing a json object instead, and provide a function that takes the json as input to create the client? That could be shared between Cordova and Electron